tools/generate-test-cases

```diff
@@ -3,84 +3,75 @@
-                image_file = fn
-        test_case["image-info"] = json.loads(subprocess.check_output(["tools/image-info", image_file]))
+format_name_dict = {
+    "partitioned-disk": "disk.img",
+    "ext4-filesystem": "filesystem.img",
+    "tar": "root.tar.xz",
+    "openstack": "disk.qcow2",
+    "qcow2": "disk.qcow2",
+    "vhd": "disk.vhd",        
+    "vmdk": "disk.vmdk",
+    "ami": "image.raw.xz",
+}
 
-    return test_case
 
+def main(distro, arch, store, output):
+    test_case = []
+    for output_format, boot in format_boot_dict.items():
+        test_case_request = {
+            "boot": {
+                "type": boot
+            },
+            "compose-request": {
+                "distro": distro,
+                "arch": arch,
+                "output-format": output_format,
+                "filename": format_name_dict[output_format],
+                "blueprint": {
+                    "name": output_format + "-boot-test",
+                    "description": "Image for boot test",
+                    "packages": [],
+                    "modules": [],
+                    "groups": [],
+                    "customizations": {
+                        "user": [
+                            {
+                                "name": "redhat",
+                                "key": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC61wMCjOSHwbVb4VfVyl5sn497qW4PsdQ7Ty7aD6wDNZ/QjjULkDV/yW5WjDlDQ7UqFH0Sr7vywjqDizUAqK7zM5FsUKsUXWHWwg/ehKg8j9xKcMv11AkFoUoujtfAujnKODkk58XSA9whPr7qcw3vPrmog680pnMSzf9LC7J6kXfs6lkoKfBh9VnlxusCrw2yg0qI1fHAZBLPx7mW6+me71QZsS6sVz8v8KXyrXsKTdnF50FjzHcK9HXDBtSJS5wA3fkcRYymJe0o6WMWNdgSRVpoSiWaHHmFgdMUJaYoCfhXzyl7LtNb3Q+Sveg+tJK7JaRXBLMUllOlJ6ll5Hod root@localhost"
+                            }
+                        ]
+                    }
+                }
+            }
+        }
+        test_case = json.loads(subprocess.check_output(["tools/generate-test-case", store], input=json.dumps(test_case_request), encoding="utf-8"))
+        name = distro.replace("-", "_") + "-" + arch + "-" + output_format.replace("-", "_") + "-boot.json"
+        file_name = output + "/" + name
+        with open(file_name, 'w') as case_file:
+            json.dump(test_case, case_file, indent=2)
+    return
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description="Generate test cases")
-    parser.add_argument("store", metavar="DIRECTORY", type=os.path.abspath, help="path to the osbuild store")
+    parser.add_argument("--distro", help="distribution for test cases", required=True)
+    parser.add_argument("--arch", help="architecture for test cases", required=True)
+    parser.add_argument("--store", metavar="STORE_DIRECTORY", type=os.path.abspath, help="path to the osbuild store", required=True)
+    parser.add_argument("--output", metavar="OUTPUT_DIRECTORY", type=os.path.abspath, help="path to the osbuild store", required=True)
```

Comment:
`help` parameter seems to be copy-pasted from the previous line

Reply:
Thanks for pointing this out. I have updated the help text.